### DISCUSSION TITLE
How to use this plugin?

### DISCUSSION THREAD

**Discussion by ReazenML** [Arisego/UnrealLive2D#21] [Q&A] [answered]
Hi there! I'm aware that this plugin isn't production-ready, but I really want to experiment with it a bit. I'm quite new to Unreal Engine, and I'm confused about how it works. Is it possible to get a small, step-by-step guide on how to set it up and add simple models myself?
I'd appreciate any help! :D

**Comment by Arisego** [comment, marked as answer]
You can try this plugin with this sample repository: https://github.com/Arisego/Live2DTest. It's all configured and ready to use.
